### ISSUE TITLE
v1.16.0 cannot init service from manifest

### ISSUE THREAD

**Issue by ghost** [aws/copilot-cli#3434]
1. Initialize a test-app.
```
copilot app init test-app
```

2. Initialize a Load Balanced Web Service.
```
copilot svc init -n test-svc -t "Load Balanced Web Service" -i "TEST-IMAGE" --port 80
```

3. Delete the service.
```
copilot svc delete -n test-svc
```

4. Now I have a manifest file.
```
# tree -a
.
`-- copilot
    |-- .workspace
    `-- test-svc
        `-- manifest.yml
```

5. Initialize service from manifest file
```
# copilot svc init -n test-svc
Note: It's best to run this command in the root of your workspace.

  Which service type best represents your service's architecture?  [Use arrows to move, type to filter, ? for more help]
  > Request-Driven Web Service  (App Runner)
    Load Balanced Web Service   (Internet to ECS on Fargate)
    Backend Service             (ECS on Fargate)
    Worker Service              (Events to SQS to ECS on Fargate)
```

It does not read service type from manifest file.
v1.15.0 works fine.

Workaround is to specify service type.
```
svc init -n test-svc -t "Load Balanced Web Service"
```

**Comment by iamhopaul123** [comment]
Hi @hiranoq. Sorry for the churn. I think the request makes sense to us and I just sent out a fix for that. After this change, you would be able to _only_ specify the name to create a service with existing local manifest. Note that the change will be included in our next release which is planned next week at this moment. Thank for reporting this issue!

**Comment by huanjani** [comment]
The fix is now released in v1.17.0: https://github.com/aws/copilot-cli/releases/tag/v1.17.0
Thank you for bringing this bug to our attention!
